src/main/scala/org/broadinstitute/dsde/workbench/avram/util/RestClient.scala

```diff
@@ -3,15 +3,19 @@ package org.broadinstitute.dsde.workbench.avram.util
 import java.net.URI
 import java.util.logging.Logger
 
-import com.softwaremill.sttp.{HttpURLConnectionBackend, Request, Uri, sttp}
+import cats.effect._
+import com.softwaremill.sttp._
+import com.softwaremill.sttp.asynchttpclient.cats.AsyncHttpClientCatsBackend
 
 /**
   * Helpful stuff for making rest calls using sttp (https://github.com/softwaremill/sttp).
   */
 trait RestClient {
   private val log: Logger = Logger.getLogger(getClass.getName)
 
-  implicit val sttpBackend = HttpURLConnectionBackend() // TODO: consider refactoring to allow use of SttpBackendStub in tests
+  //implicit val sttpBackend = HttpURLConnectionBackend() // TODO: consider refactoring to allow use of SttpBackendStub in tests
+
+  implicit val sttpBackend = AsyncHttpClientCatsBackend[IO]()
```

Comment:
Makes sttp return an `IO[A]` which fits into our monad stack instead of an `Id[A]`.